xml/operations-dns-designate_initialconfig.xml

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entities.ent"> %entities;
+]>
+<section xml:id="DesignateInitialConfig"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>&desig; Initial Configuration</title>
+ <para>
+  After the <phrase vendor="hpe">"&hpe; Helion
+  &ostack;</phrase><phrase vendor="suse">&suse; &ostack;</phrase> installation
```

Comment:
please use the much simpler `&productname;` for this -- this is always and automatically right and save you a lot of typing.